app/services/candidate_interface/find_changed_apply_again_applications.rb

```diff
@@ -0,0 +1,48 @@
+module CandidateInterface
+  class FindChangedApplyAgainApplications
+    def all_forms
+      apply_again_forms
+    end
+
+    def changed_forms
+      apply_again_forms.find_each.lazy.select do |application_form|
+        changed?(application_form)
+      end
+    end
+
+    def all_candidate_count
+      all_forms.map(&:candidate_id).uniq.count
```

Comment:
As this is a relation, could we `pluck` rather than `map`?

Reply:
Yes good point. I've gone with `select(:candidate_id).distinct.count` so that we don't load anything into memory other than the end result. Does that make sense?